src/Magnum/DartIntegration/Test/DartSkeletonTest.cpp

```diff
@@ -0,0 +1,267 @@
+    properties.mSpringStiffnesses = Eigen::Vector3d::Constant(default_stiffness);
+    properties.mDampingCoefficients = Eigen::Vector3d::Constant(default_damping);
+
+    dart::dynamics::BodyNodePtr bn = pendulum->createJointAndBodyNodePair<dart::dynamics::BallJoint>(
+                                                 nullptr, properties, dart::dynamics::BodyNode::AspectProperties(name))
+                                         .second;
+
+    // Make a shape for the Joint
+    const double& R = default_width;
+    std::shared_ptr<dart::dynamics::EllipsoidShape> ball(
+        new dart::dynamics::EllipsoidShape(sqrt(2) * Eigen::Vector3d(R, R, R)));
+    auto shapeNode = bn->createShapeNodeWith<dart::dynamics::VisualAspect>(ball);
+    shapeNode->getVisualAspect()->setColor(dart::Color::Blue());
+
+    // Set the geometry of the Body
+    setGeometry(bn);
+
+    return bn;
+}
+
+inline dart::dynamics::BodyNode* addBody(const dart::dynamics::SkeletonPtr& pendulum, dart::dynamics::BodyNode* parent, const std::string& name) {
+    // Set up the properties for the Joint
+    dart::dynamics::RevoluteJoint::Properties properties;
+    properties.mName = name + "_joint";
+    properties.mAxis = Eigen::Vector3d::UnitY();
+    properties.mT_ParentBodyToJoint.translation() = Eigen::Vector3d(0, 0, default_height);
+    properties.mRestPositions[0] = default_rest_position;
+    properties.mSpringStiffnesses[0] = default_stiffness;
+    properties.mDampingCoefficients[0] = default_damping;
+
+    // Create a new BodyNode, attached to its parent by a RevoluteJoint
+    dart::dynamics::BodyNodePtr bn = pendulum->createJointAndBodyNodePair<dart::dynamics::RevoluteJoint>(
+                                                 parent, properties, dart::dynamics::BodyNode::AspectProperties(name))
+                                         .second;
+
+    // Make a shape for the Joint
+    const double R = default_width / 2.0;
+    const double h = default_depth;
+    std::shared_ptr<dart::dynamics::CylinderShape> cyl(new dart::dynamics::CylinderShape(R, h));
+
+    // Line up the cylinder with the Joint axis
+    Eigen::Isometry3d tf(Eigen::Isometry3d::Identity());
+    tf.linear() = dart::math::eulerXYZToMatrix(
+        Eigen::Vector3d(90.0 * M_PI / 180.0, 0, 0));
+
+    auto shapeNode = bn->createShapeNodeWith<dart::dynamics::VisualAspect>(cyl);
+    shapeNode->getVisualAspect()->setColor(dart::Color::Blue());
+    shapeNode->setRelativeTransform(tf);
+
+    // Set the geometry of the Body
+    setGeometry(bn);
+
+    return bn;
+}
+
+namespace Magnum { namespace DartIntegration { namespace Test {
+
+namespace {
+
+    using namespace Math::Literals;
+
+    typedef SceneGraph::Object<SceneGraph::MatrixTransformation3D> Object3D;
+    typedef SceneGraph::Scene<SceneGraph::MatrixTransformation3D> Scene3D;
+
+}
+
+struct DartSkeletonTest : TestSuite::Tester {
+    explicit DartSkeletonTest();
+
+    void test();
+    void test_urdf();
+    void test_texture();
```

Comment:
Coding style: no underscores, please :) Also you can drop the `test` prefix and have just `urdf()` and `texture()` there.